ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml

```diff
@@ -29,6 +29,19 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
+- always_run: false
```

Comment:
Why we still have jobs in openshift-test-private? Can we have a new folder?

Reply:
This is a demo job. We should have the job under the `rosa` project.

Reply:
Using `periodic-ci-openshift-rosa-master-e2e-rosa-profile-demo` instead of